package.json

```diff
@@ -14,7 +14,8 @@
     "sketch": ">=3.0"
   },
   "devDependencies": {
-    "eslint": "^4.4.1",
+    "eslint": "^4.10.0",
+    "eslint-plugin-import": "^2.8.0",
```

Comment:
This shouldn't be needed because `frontend-tools-configs` is responsible for pulling all linting dependencies. However, I stated testing and found out that eslint setup was broken (it was linting wrong folders, `frontend-tools-configs` haven't had locked version). I just fixed it - please try again.